identity_credential/src/presentation/jwp_presentation_builder.rs

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020-2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::error::Error;
+use crate::error::Result;
+use jsonprooftoken::jwp::header::PresentationProtectedHeader;
+use jsonprooftoken::jwp::issued::JwpIssued;
+use jsonprooftoken::jwp::presented::JwpPresentedBuilder;
+
+/// Used to construct a JwpPresentedBuilder and handle the selective disclosure of attributes
```

Comment:
The comment there is kinda strange as it's more intended for us developers of identity.rs rather than its users. Through the public API of JwpPresentationBuilder you can only conceal fields of `credentialSubject` and `evidence` so there's no way for user to conceal the credential type for instance.